File: server/API/Common/Enum.cs
using System.ComponentModel;

namespace API.Common
{
    public enum ProjectPhase
    {
        None,

        [Description("Giai đoạn mở bán đầu tiên")]
        MoBan,

        [Description("Giai đoạn chuẩn bị hoàn thiện")]
        HoanThien,

        [Description("Giai đoạn bàn giao các hạng mục")]
        BanGiao
    }

    public enum ProjectType
    {
        None,

        [Description("Bất động sản đô thị")]
        DoThi,

        [Description("Bất động sản du lịch")]
        DuLich,

        [Description("Bất động sản công nghiệp")]
        CongNghiep
    }

    public enum ItemType
    {
        None,
        User,
        Project,
        Content,
        Event,
        Partner
    }

    public enum ContentType
    {
        None,

        [Description("Bài viết giới thiệu")]
        GioiThieu,

        [Description("Bài viết cập nhật tiến độ")]
        CapNhatTienDo,

        [Description("Bài viết phản hồi")]
        Review
    }

    public enum EventType
    {
        None,

        [Description("Diễn ra định kỳ")]
        DinhKy,

        [Description("Diễn ra hàng nằm")]
        HangNam,

        [Description("Diễn ra một lần duy nhất")]
        DuyNhat
    }
    public enum EventStatus
    {
        None,

        [Description("Đang lên kế hoạch")]
        LenKeHoach,

        [Description("Sắp diễn ra")]
        SapDienRa,

        [Description("Đang diễn ra")]
        DangDienRa,

        [Description("Đã kết thúc")]
        KetThuc
    }

    public enum SubcribeType
    {
        None,
        [Description("Nhận mail cập nhật tiến độ")]
        Update,
        [Description("Nhận mail khi có dự án mới")]
        Create,
        [Description("Nhận mail khi có dự án mới lẫn cập nhật tiến độ")]
        All
    }
}

File: client/projects/management/src/app/shared/core/Enum.ts
export const ProjectType = [
    { id: 1, name: 'Bất động sản đô thị' },
    { id: 2, name: 'Bất động sản du lịch' },
    { id: 3, name: 'Bất động sản công nghiệp' }
];

export const ProjectPhase = [
    { id: 1, name: 'Giai đoạn mở bán đầu tiên' },
    { id: 2, name: 'Giai đoạn chuẩn bị hoàn thiện' },
    { id: 3, name: 'Giai đoạn bàn giao các hạng mục' }
];
export const ContentType = [
    { id: 1, name: 'Bài viết giới thiệu' },
    { id: 2, name: 'Bài viết cập nhật tiến độ' },
    { id: 3, name: 'Bài viết phản hồi' }
];
export const EventType = [
    { id: 1, name: 'Diễn ra định kỳ' },
    { id: 2, name: 'Diễn ra hàng năm' },
    { id: 3, name: 'Diễn ra một lần duy nhất' }
];
export const EventStatus = [
    { id: 1, name: 'Đang lên kế hoạch' },
    { id: 2, name: 'Sắp diễn ra' },
    { id: 3, name: 'Đang diễn ra' },
    { id: 4, name: 'Đã kết thúc' }
];

export enum ProjectTypeEnum {
    None,
    Urban,
    Tourism,
    Industrial
}

export enum ItemType {
    None,
    User,
    Project,
    Content,
    Event,
    Partner
}